core/src/main/scala/com/evolutiongaming/kafka/flow/KeyFlowOf.scala

```diff
@@ -3,15 +3,18 @@ package com.evolutiongaming.kafka.flow
 import cats.Monad
 import cats.effect.{Ref, Resource}
 import com.evolutiongaming.kafka.flow.persistence.Persistence
+import com.evolutiongaming.kafka.flow.registry.EntityRegistry
 import com.evolutiongaming.kafka.flow.timer.{TimerContext, TimerFlowOf}
 
 trait KeyFlowOf[F[_], S, A] {
 
   def apply(
+    key: KafkaKey,
```

Comment:
This one is a bit trickier. It's probably possible to make something like this:
```scala
@deprecated
trait KeyFlowOf[F[_], S, A] extends KeyFlowOf2[F, S, A] {

  def apply(
    context: KeyContext[F],
    persistence: Persistence[F, S, A],
    timers: TimerContext[F],
    additionalPersist: AdditionalStatePersist[F, S, A],
  ): Resource[F, KeyFlow[F, A]]

  final def apply(
    key: KafkaKey,
    context: KeyContext[F],
    persistence: Persistence[F, S, A],
    timers: TimerContext[F],
    additionalPersist: AdditionalStatePersist[F, S, A],
    registry: EntityRegistry[F, KafkaKey, S],
  ): Resource[F, KeyFlow[F, A]] = apply(fakeKafkaKey, context, persistence, timers, additionalPersist, EntityRegistry.noop)

}

trait KeyFlowOf2[F[_], S, A] {

  def apply(
    key: KafkaKey,
    context: KeyContext[F],
    persistence: Persistence[F, S, A],
    timers: TimerContext[F],
    additionalPersist: AdditionalStatePersist[F, S, A],
    registry: EntityRegistry[F, KafkaKey, S],
  ): Resource[F, KeyFlow[F, A]]

}
```

and use KeyFlowOf2 in the library code